force-app/main/default/lwc/geGatewaySelectWidget/geGatewaySelectWidget.js

```diff
@@ -18,21 +18,28 @@ import psShowPaymentMethods from '@salesforce/label/c.psShowPaymentMethods';
 import psUnableToConnect from '@salesforce/label/c.psUnableToConnect';
 import RD2_Credit_Card_Payment_Method_Label from '@salesforce/label/c.RD2_Credit_Card_Payment_Method_Label';
 import GeGatewaySettings from 'c/geGatewaySettings';
-import { fireEvent } from 'c/pubsubNoPageRef';
+import { fireEvent, registerListener } from 'c/pubsubNoPageRef';
 import { isEmpty, showToast } from 'c/utilCommon';
 
+const TEMPLATE_CSS_CLASS = 'template-top';
+const GIFT_ENTRY_CSS_CLASS = 'gift-entry-top';
+const TOKENIZE_ID = 'TOKENIZE';
+const MANAGEMENT_ID = 'MANAGEMENT';
```

Comment:
Are these constants used to indicate which page this change is happening on? If so, can we update the variable name and or string to indicate that? Even `parentId` feels like it should be either a record ID or Elevate ID, not a constant string value.